x-pack/legacy/plugins/snapshot_restore/public/app/services/http/http.ts

```diff
@@ -3,18 +3,15 @@
  * or more contributor license agreements. Licensed under the Elastic License;
  * you may not use this file except in compliance with the Elastic License.
  */
-class HttpService {
```

Comment:
I was the one that suggested the HttpService to jen. Is there a good reason to remove it here?

My point is that I prefer to encapsulate the variables in a class instance instead of relying on the cached variable from the import system.

I know that your changes here align with other places where we do it, but the Platform team is leaning more toward the use of Classes for those use cases.

Reply:
fwiw I prefer using a Class instance for these services

Reply:
Thanks for the explanations Seb and Jen! I wasn't aware that modules weren't guaranteed to be singletons and that exporting class instances was a workaround for that. I think your reasoning makes sense Jen. I'll revert these types of changes.